### PR TITLE
fix: add api call to access ms-finance

### PR DESCRIPTION
**Pull request by Willbrighte**
* add api call to getApplicationId
* add api call to addJointApplicant

### CODE REVIEW

File: src/FinanceApi.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+class FinanceApi extends \BrighteCapital\Api\AbstractApi
+{
+    public const PATH = '/finance';
+
+    public function getApplicationId(string $remoteId): ?string
+    {
+        $response = $this->brighteApi->get(self::PATH . '/applications/account/' . $remoteId, '', [], self::PATH);
+
+        if ($response->getStatusCode() !== StatusCodeInterface::STATUS_OK) {
+            var_dump("HElala");
```
Comment (chrisyoungbrighte) [inline src/FinanceApi.php:18]: please remove this :)

File: src/FinanceApi.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+            return null;
+        }
+
+        $result = json_decode((string) $response->getBody());
+
+        return $result->id ?? null;
+    }
+
+    public function addJointApplicant(array $user, string $applicationId): ?string
```
Comment (chrisyoungbrighte) [inline src/FinanceApi.php:29]: suggestion: change `user` to applicant. they are not a user yet

File: src/FinanceApi.php (hunk 1 above)
Comment (Willbrighte) [reply src/FinanceApi.php]: yeah, just saw it, thanks @chrisyoungbrighte.

File: src/FinanceApi.php (hunk 2 above)
Comment (Willbrighte) [reply src/FinanceApi.php:28]: noted, will update.

**Review by chrisyoungbrighte — APPROVED** [review]

**Review by fahadbrighte — APPROVED** [review]

**Pull request merged**